apps/src/templates/studioHomepages/SetUpSections.story.jsx

```diff
@@ -1,30 +1,31 @@
 import React from 'react';
 import SetUpSections from './SetUpSections';
 import {action} from '@storybook/addon-actions';
+import {reduxStore} from '@cdo/storybook/decorators';
+import {Provider} from 'react-redux';
 
-export default storybook =>
-  storybook
-    .storiesOf('Homepages/Teachers/SetUpSections', module)
-    .withReduxStore()
-    .addStoryTable([
-      {
-        name: 'Set Up Message for Sections - no sections yet',
-        description: `Information box if the teacher doesn't have any sections yet`,
-        story: () => (
-          <SetUpSections
-            beginEditingSection={action('beginEditingSection')}
-            hasSections={false}
-          />
-        ),
-      },
-      {
-        name: 'Set Up Message for Sections - already has section',
-        description: `Information box if the teacher does have sections already`,
-        story: () => (
-          <SetUpSections
-            beginEditingSection={action('beginEditingSection')}
-            hasSections={true}
-          />
-        ),
-      },
-    ]);
+export default {
+  title: 'SetUpSections',
+  component: SetUpSections,
```

Comment:
Are we still going to need this once @hannahbergam is done with section setup clean up?

Reply:
These parts of the redux file make my brain hurt. We are still using a lot of the same entry points so its been hard for me to parse through what we still need. I'd say for now we can keep this story because according to [this figma](https://www.figma.com/file/fe70qoiGWnV78j9MKfAEdo/Teacher-Experience-UX%2FUI-Improvements?type=design&node-id=1549-18311&t=8JZLEIXKMMwy5kMh-0), we will reuse this component in the new flow!